runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -212,6 +212,14 @@ public Dimension render(Graphics2D graphics, java.awt.Point parent)
 						continue;
 					}
 
+					if (config.showHighlightedOnly())
```

Comment:
Maybe this should be moved to condition where there is check for hidden items on line 157.